docker-entrypoint.sh

```diff
@@ -22,12 +22,15 @@
 JOB_MANAGER_RPC_ADDRESS=${JOB_MANAGER_RPC_ADDRESS:-$(hostname -f)}
 
 drop_privs_cmd() {
-    if [ -x /sbin/su-exec ]; then
+    if [ $(id -u) != 0 ]; then
```

Comment:
would be better to check whoami = flink ?
because it will fail if the user is not root or flink

Reply:
I'm not sure that's correct: while the files are owned by the `flink` user, an arbitrary user can still read the files and thus execute Flink.

Additionally, some platforms such as OpenShift actually randomize the UID, meaning the image should support running as arbitrary (non-9999) UIDs.

If you want to test this yourself, you can try it with an image I built from this branch:
```
$ docker run --rm --user 1234 -i plucas/docker-flink-pr67 id
$ docker run --rm --user 1234 -i -p 8081:8081 plucas/docker-flink-pr67 jobmanager
```

Reply:
plucas/docker-flink-pr67 with k8s and a security context (user 9999) works!